core/src/utils/helpers.ts

```diff
@@ -424,3 +424,10 @@ export const getNextSiblingOfType = <T extends Element>(element: Element): T | n
   }
   return null;
 };
+
+/**
+ * Checks input for usable number. Not NaN and not Infinite.
+ */
+export const isSafeNumber = (input: unknown): input is number => {
```

Comment:
Should this be `as` not `is`?

```suggestion
export const isSafeNumber = (input: unknown): input as number => {
```

Reply:
No, this is a typescript type predicate for return types. It tells typescript that if the result is true, it can safely assume the input is a number from then on out. So if you do `if (isSafeNumber(abc)) abc++;` then typescript knows that's fine because you checked and told typescript it's safe to assume it's a number now.